android/build.gradle

```diff
@@ -192,3 +192,58 @@ dependencies {
         }
     }
 }
+
+def isScreensExampleApp() {
+    return project.hasProperty('isScreensExampleApp') && project.property('isScreensExampleApp')
+}
+
+task copyCodegenArtifacts(type: Copy) {
+    group 'After build tasks'
+    description 'Tasks which copy codegen artifacts to paper architecture'
+    
+    if (!isScreensExampleApp()) {
+        return
+    }
+
+    dependsOn tasks.generateCodegenArtifactsFromSchema
+
+    def absoluteCodegenArtifactsPaperDestination = "${project.rootDir}/../../${project.property('codegenArtifactsPaperDestination')}"
+    def absoluteCodegenArtifactsSource = "${project.rootDir}/../../${project.property('codegenArtifactsSource')}"
+
+    def existingFiles = fileTree(absoluteCodegenArtifactsPaperDestination).matching {
+        include '**/*.java'
+    }
+
+    def generatedFiles = fileTree(absoluteCodegenArtifactsSource).matching {
+        include '**/*.java'
+    }
+
+    def existingFilesMap = [:]
+
+    existingFiles.forEach { existingFile ->
+        existingFilesMap[existingFile.name] = 1
+    }
+
+    generatedFiles.forEach { generatedFile -> 
+        if (!existingFilesMap.containsKey(generatedFile.name)) {
+            println generatedFile.name + " not found in paper dir"
+        }
+    }
+
+    if (existingFiles.size() <= 0) {
+        throw new Exception('Paper destination with codegen interfaces is empty.')
+    }
```

Comment:
Why is this an error? Also it would be nice if error messages suggested an action to take (another nitpick)

Reply:
I changed it to warning, cause this may indicate that the paper destination folder is incorrect. Changed in https://github.com/software-mansion/react-native-screens/pull/2168/commits/bd65c0c7bb6d6a12facbf79e40c1a04b89aebb4f